### ISSUE TITLE
Post Translation only partially working

### ISSUE THREAD

**Issue by rocket357** [mastodon/mastodon#20795]
### Steps to reproduce the problem

1. Set post language to ES
2. Enter post text: "Yo quiero hablar espanol.  (probando un boton)"  (should read: "I want to speak spanish (testing a button)"
3. Translate the post.
...

### Expected behaviour

Expected output:  "I want to speak spanish (testing a button)"

### Actual behaviour

Actual output:  "Yo quiero hablar espanol.  (testing a button)"

### Detailed description

This was tested on mastodon.lol and verified in discussion with the admin (@nathan@mastodon.lol).  Only the portion in the parenthesis is translated.

### Specifications

Mastodon 4.0.2
Web interface (Chrome 107.0.5304.106 (Official Build) (64-bit) on Windows 10.0.19044 Build 19044)

**Comment by vmstan** [comment]
This appears to be an issue with LibreTranslate translation engine itself, the behavior does not reproduce on DeepL enabled instances.

Pasting this field into LT directly can demonstrate.

<img width="924" alt="image" src="https://github.com/mastodon/mastodon/assets/3002053/0e722e33-e60d-4559-8c82-1cdf168e0514">

**Comment by vmstan** [comment]
After the using accents it's able to function correctly.

<img width="924" alt="image" src="https://github.com/mastodon/mastodon/assets/3002053/17dc3ca7-5273-4413-9c49-cc93ad77553c">

This appears to be a quirk about LT, and not a bug in Mastodon.

**Comment by rocket357** [comment]
Hi,

How does the "actual output" above correlate to the LT direct test you supplied?  (Yes, LT does get that incorrect...but that's *not* what I was pointing out in this bug... the issue is that the first half of the phrase wasn't even translated, regardless if it had the accents or not.).

But in all fairness, a lot has happened in the year since I opened this bug.  Nathan shut down mastodon.lol and I've moved on, so...I guess this is fairly irrelevant now.
